cse-lib/data/src/main/java/com/farao_community/farao/cse/data/cnec/CracResultsHelper.java

```diff
@@ -182,19 +184,20 @@ public Map<String, MergedCnec> getMergedCnecs(String contingencyId) {
                         throw new CseDataException("Couldn't find Cnec type in cnec Id : " + flowCnec.getId());
                 }
             }
-            CnecCommon cnecCommon = makeCnecCommon(cnecName, branchCnecCreationContext.getNativeBranch(),
+            CnecCommon cnecCommon = makeCnecCommon(flowCnec, branchCnecCreationContext.getNativeBranch(),
```

Comment:
Is there a possibility for `flowCnec` to be null at this point?

Reply:
I have got to ask how to treat this then....

Reply:
confirmed by @murgeyseb shouldn't happen